vip_spec.xsd

```diff
@@ -579,6 +580,30 @@ reference elements for polling location
     </xs:restriction>
   </xs:simpleType>
 
+  <xs:simpleType name="DistrictType">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="city" />
+      <xs:enumeration value="congressional" />
+      <xs:enumeration value="council" />
```

Comment:
> but for which we don't have sufficiently unique enum values

Also, how can you ever know in advance that the enum values are sufficiently unique?  For example, I'm guessing there is probably more than one kind of "judicial" district in some parts of the country.  In California, the state has six Courts of Appeal districts, but maybe some parts of the country also have more localized judicial districts.  The SF Bay Area also has BART districts (a type of transportation district), which I guess would be classified as an under in this scheme.  California also has four "Board of Equalization" districts, which I guess would be another "other."